src/game_api/script/usertypes/entities_logical_lua.cpp

```diff
@@ -230,8 +230,24 @@ void register_usertypes(sol::state& lua)
 
     lua.new_usertype<LimbAnchor>(
         "LimbAnchor",
+        "anchor_x",
+        &LimbAnchor::anchor_x,
+        "anchor_y",
+        &LimbAnchor::anchor_y,
+        "move_origin_x",
+        &LimbAnchor::move_origin_x,
+        "move_origin_y",
+        &LimbAnchor::move_origin_y,
+        "move_destination_x",
+        &LimbAnchor::move_destination_x,
+        "move_destination_y",
+        &LimbAnchor::move_destination_y,
         "move_timer",
-        &LimbAnchor::move_timer,
+        &LimbAnchor::elapsed_move_time,
+        "elapsed_move_time",
+        &LimbAnchor::elapsed_move_time,
```

Comment:
Can you make those renames  `/// NoDoc` ?
It's pointless to have them in the doc
have to either convert to the other style of declaring new lua type or use `union` in the class itself

Also would be great if you could re-generate the API doc, (I should probably make like github workflow for it as well), you have to run cmake target called `update_auto_docs_overlunky` (it just runs two python scripts in specific working directory)